snapcraft/plugins/meson.py

```diff
@@ -45,20 +52,50 @@ def schema(cls):
             "items": {"type": "string"},
             "default": [],
         }
+        schema["properties"]["meson-version"] = {"type": "string", "default": ""}
```

Comment:
This default doesn't match up with the docs above. Should the docs say "if unspecified, the latest will be used"?

Reply:
I'll fix, thanks